lib/rules/prefer-wait-for.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+const { getDocsUrl } = require('../utils');
+
+const DEPRECATED_METHODS = ['wait', 'waitForElement', 'waitForDomChange'];
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Use `waitFor` instead of deprecated wait methods',
+      category: 'Best Practices',
+      recommended: false,
+      url: getDocsUrl('prefer-wait-for'),
+    },
+    messages: {
+      preferWaitForMethod:
+        '`{{ methodName }}` is deprecated in favour of `waitFor`',
+      preferWaitForImport: 'import `waitFor` instead of deprecated async utils',
+    },
+    fixable: 'code',
+    schema: [],
+  },
+
+  create: function(context) {
+    const importNodes = [];
+    const waitNodes = [];
+
+    const reportImport = node => {
+      context.report({
+        node: node,
+        messageId: 'preferWaitForImport',
+        fix(fixer) {
+          const fixers = [];
+          let waitForImported = false;
+
+          node.specifiers.forEach((specifier, idx, specifiers) => {
+            if (DEPRECATED_METHODS.includes(specifier.imported.name)) {
+              if (waitForImported) {
```

Comment:
Should we take into account that `waitFor` is already imported (manually), before running this rule?

Reply:
Yeah, that's a good point. It's easier to check here, I'll add that later!

Reply:
Solved with the new import fix code